src/overlays/actors/ovl_Bg_Tobira01/z_bg_tobira01.c

```diff
@@ -15,8 +15,8 @@ void BgTobira01_Init(Actor* thisx, PlayState* play);
 void BgTobira01_Destroy(Actor* thisx, PlayState* play);
 void BgTobira01_Update(Actor* thisx, PlayState* play);
 void BgTobira01_Draw(Actor* thisx, PlayState* play);
+void BgTobira01_Listen(BgTobira01* this, PlayState* play);
```

Comment:
What is this function? Was it `BgTobira01_Action` previously? If it was, you don't need to forward declare since `BgTobira01_Action` is the first function